lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,12 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
+    def update_destination(destination_branch, remote = 'origin')
       origin_branch = current_branch
       run_command_with_label "git checkout #{destination_branch}"
-      run_command_with_label "git pull origin #{destination_branch}"
+      run_command_with_label "git pull #{remote} #{destination_branch}"
       run_command_with_label "git checkout #{origin_branch}"
+      run_command_with_label "git rebase #{destination_branch}"
```

Comment:
let's move this into a new function that can encapsulate the intent better:

```ruby
def update_feature_branch(branch_name = current_branch)
# first update the base branch(options = {})
base_branch = options[:base_branch] || 'master'
feature_branch = options[:feature_branch] || current_branch
remote = options[:remote] || 'origin'
update_destination(base_branch, remote: remote)
# then update the feature branch in case there are multiple authors and has remote changes
update_current_branch
# then rebase on the base branch to bring it up to date
run_command_with_label "git rebase #{branch_name}"
end
```